app/mailers/notify_mailer.rb

```diff
@@ -35,9 +35,17 @@ def new_mention_email
 
     @mentioner = User.find(@mention.mentionable.user_id)
     @mentionable = @mention.mentionable
+
+    # Articles are colloquially referred to as "posts", so we need to do some conditional renaming here.
+    @mentionable_type = if @mention.mentionable_type == "Article"
```

Comment:
**suggestion (non-blocking):** How about adding a `MentionDecorator` and encapsulating this behavior there? We may need this eventually in other places and I'd rather see it properly done via a decorator than with a helper.